containers/runner/README.md

```diff
@@ -44,7 +49,7 @@ curl -L https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/
 
 Run the test:
 ```
-podman run --env KSTESTS_TEST=keyboard -v ${VOLUME_DIR}:/opt/kstest/data:z --name last-kstest --device=/dev/kvm kstest-runner
+podman run --env KSTESTS_TEST=keyboard -v ${VOLUME_DIR}:/opt/kstest/data:z --name last-kstest --device=/dev/kvm rhinstaller/kstest-runner
```

Comment:
Should it be with the ``:latest`` tag here?

Reply:
:latest is the default if you don't specify it. As this is a command that the user has to type/run, I thought I'd aim for brevity. Unlike anaconda-ci, we will only work with `:latest` for this container (unless you explicitly want to try an earlier one). So if you prefer, I'm ok with adding the `:latest` explicitly, I'm just not sure if that was a question or a request?